support/cas-server-support-saml-idp-metadata/src/test/java/org/apereo/cas/support/saml/services/idp/metadata/cache/resolver/JsonResourceMetadataResolverTests.java

```diff
@@ -49,4 +50,14 @@ public void verifyResolverResolves() throws Exception {
         val resolved = metadataResolver.resolveSingle(new CriteriaSet(new EntityIdCriterion("https://example.org/saml")));
         assertNotNull(resolved);
     }
+
+    /**
+     * Make sure default file:/etc/cas/saml URI is processed.
+     */
+    @Test
+    public void verifyResolverResolvesWithFileUri() throws Exception {
+        val props = new SamlIdPProperties();
+        val resolver = new JsonResourceMetadataResolver(props, openSamlConfigBean);
```

Comment:
It would be best to actually set the property value to always make sure the test is accurate, in cas the default ever changes.